RelationalAI/AccessToken.cs

```diff
@@ -15,29 +15,38 @@
  */
 
 using System;
+using Newtonsoft.Json;
 
 namespace RelationalAI
 {
-    public class AccessToken
+    public class AccessToken : Entity
     {
-        private readonly DateTime _createdOn;
+        [JsonProperty("expires_in")]
         private int _expiresIn;
 
-        public AccessToken(string token, int expiresIn)
+        public AccessToken(string token, int expiresIn, string scope)
         {
             Token = token;
-            ExpiresIn = expiresIn;
-            _createdOn = DateTime.Now;
+            _expiresIn = expiresIn;
+            Scope = scope;
+            CreatedOn = new DateTimeOffset(DateTime.Now).ToUnixTimeSeconds();
```

Comment:
`AccessToken` will be serialized to disk and we need to make sure that all fields have unique format across all SDKs in case another SDK will use the cached `AccessToken`